.azure-pipelines/code-oob.yml

```
@@ -1,70 +1,44 @@
 parameters:
-- name: TargetBranch
-  displayName: Branch this module will generated to
-  type: string
-  default: main
 - name: ServiceName
-  displayName: Service to OOB release
+  displayName: OOB release Service
   type: string
   default: Databricks
 - name: Preview
-  displayName: Is preview version
+  displayName: Is preview version(Including < 1.0.0)

```

Here we can be more clear
```suggestion
  displayName: Is preview version? (version is "x.y.z-preview" or major version is 0)
```